playbooks/pre_cloud.yaml

```
@@ -27,5 +27,5 @@
         mode: "0600"
 
     - name: Clean up any lingering resources from previous run
-      ansible.builtin.shell: python3 ~/Tests/cleanup.py -c {{ cloud }} --prefix _scs-
+      ansible.builtin.shell: python3 ~/Tests/cleanup.py -c {{ cloud }} --prefix _scs- --ipaddr 10.1.0.
```

```suggestion
      ansible.builtin.shell: python3 ~/Tests/cleanup.py -c {{ cloud }} --prefix _scs- --ipaddr 10.1.0.1
```

should be possible, since you changed the allocation pool to end with .199

---

This was the reason to change the allocation pool, as you correctly guessed, yes.
And then I refrained from using `10.1.0.1` over `10.1.0.`, as I was not 100% sure that we would not occasionally want the cleanup script to hit a DHCP port [.2] or gateway [.1] in this subnet in case the tear down has gone terribly wrong, so I left it with hitting the complete subnet. (OK, gateway would actually be hit ... as it matches the `startswith()` comparison as well.)